### DISCUSSION TITLE
overlapping cases

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/edit.tosdr.org#1007] [Ideas]
> tracking pixels used in service-to-user communication
> ->
> The service may use tracking pixels, web beacons, browser
> fingerprinting, and/or device fingerprinting on users.

hm, yes that's a partial overlap, but not sure how to fix that. maybe merge the first into the second, shall  we?
 
> very broad
> ->
> The copyright license maintained by the service over user data and/or
> content is broader than necessary.

yes, i  guess these should also be merged
 
also, remove the 'None' case

**Comment by JesseWeinstein** [comment]
Other cases that are too ambiguous for me to know how to judge the points linked to them include: 

https://edit.tosdr.org/cases/218 -- "This service does not track you"
https://edit.tosdr.org/cases/220 -- "The service uses your personal data to employ targeted third-party advertising"
https://edit.tosdr.org/cases/212 -- "User pseudonyms are allowed due to the service's billing policies" -- isn't this redundant with 215 "The service allows you to use pseudonyms"?

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/213 and https://edit.tosdr.org/cases/214 should probably be merged.

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/226 -- "The user is informed about security practices" ; unclear what this means or how to decide whether to approve the 14 pending points in it (right now).

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/topics/25 -- "Cookies" has 10 cases, most of which are deeply confused and duplicative, and have *75* pending points among them, basically none of which I can figure out how to rule on.

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/topics/41 -- "Personal Data" has 22 cases, and 109 pending points. Some of the cases are clear, but most overlap badly, and/or have unclear mappings to actual legal language.

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/topics/48 -- "Third Parties" has 12 cases, and 66 pending points. Some are very vague, like "Third parties may be involved in operating the service" -- *"may be"*??

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/329 (16 pending) and https://edit.tosdr.org/cases/122 (24 pending) -- seem to overlap in a confusing way with the other cases that refer to notifying users.

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/335 -- "This service does not force users into binding arbitration" is valid for every service that *does not* mention arbitration in their Terms of Service. That's ... not a very useful case (and harder to verify)!

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/297 -- "The service provides information about how they collect personal data" (with 10 pending) is WAY too broad. What information? Isn't that basically just what a Privacy Policy is?

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/227 (7 pending) and https://edit.tosdr.org/cases/228 (28 pending) are also too vague for me to tell how to judge them.

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/199 -- "The terms for this service are easy to read" (15 pending). How is this to be determined?

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/144 -- Hugo and ps have tried to clarify this case, but it's still a mess.

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/133 -- "very broad" (4 pending) This is ... um ... a "very broad" case.

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/166 "This service can share your personal information to third parties" (10 pending) -- it's unclear what this means, exactly, or what legal language would satisfy it.

**Comment by JesseWeinstein** [comment]
https://edit.tosdr.org/cases/167 "The service will not allow third parties to access your personal information without a legal basis" (4 pending) -- this overlaps with other cases in an unclear way.

**Comment by madoleary** [comment]
here's a modest proposal for how to merge some of them (i haven't gotten through all of them yet): 

- this service allows you to use pseudonyms + user pseudonyms are allowed due to the service's billing policies --> this service allows you to use pseudonyms

- you must provide your legal name, pseudonyms are not allowed + Users are not allowed to use pseudonyms, as trust and transparency between users regarding their identities is relevant to the service --> this service does not allow you to use pseudonyms

- The service uses third party tracking pixels to track social media conversion + The service may use tracking pixels, web beacons, browser fingerprinting, and/or device fingerprinting on users. + tracking pixels used in service-to-user communication --> The service may use tracking pixels, web beacons, browser fingerprinting, and/or device fingerprinting on users.

- The service uses your personal data to employ targeted third-party advertising + The service uses your personal data for advertising --> this service uses your personal data for advertising

- This service takes credit for your content + This service can use your content for all their existing and future services --> This service can use your content for all their existing and future services

- The service is transparent regarding government requests or inquiries that may involve user data. + The service promises to inform and/or notify users regarding government inquiries that may involve users' personal data --> The service is transparent regarding government requests or inquiries that may involve user data.

- The service is not transparent regarding government requests or inquiries that may involve user data. + This service reserves the right to disclose your personal information without notifying you --> The service is not transparent regarding government requests or inquiries that may involve user data.

- The service will not allow third parties to access your personal information without a legal basis + The service will resist legal requests for user information where reasonably possible -->  The service will resist legal requests for user information where reasonably possible

- The copyright license maintained by the service over user data and/or content is broader than necessary. + This service employs a broad copyright license over user content including the right to distribute through any media + very broad + This service can license user content to third parties --> The copyright license maintained by the service over user data and/or content is broader than necessary. 

- The copyright license that users grant to the service is limited for the purposes of that same service, and not third parties. + The copyright license that users grant this service is limited to the parties that make up the service's broader platform. --> The copyright license that users grant to the service is limited 

- You agree to defend, indemnify, and hold the service harmless in case of a claim related to your use of the service + defend, indemnify, hold harmless; survives termination + You are solely responsible for claims made against the service and agree to indemnify and hold harmless the service. --> You agree to defend, indemnify, and hold the service harmless in case of a claim related to your use of the service 

- only for your individual use + This service is only available for use individually and non-commercially. --> This service is only available for use individually and non-commercially.

- This service retains rights to your content even after you stop using your account + This service keeps a license on user-generated content even after users close their accounts. + This service will continue using anonymized user-generated content even after erasure of personal information

- When the service wants to change its terms, users are notified a month or more in advance + When the service wants to change its terms, users are notified a week or more in advance. --> When the service wants to change its terms, users are notified in advance
